Please update dependencies

I have installed the latest i18next (v 22.5.0) and the installation of this module complains:
```
D:> npm i i18next-cli-language-detector
npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR! 
npm ERR! While resolving: bib2web@0.2.1
npm ERR! Found: i18next@22.5.0
npm ERR! node_modules/i18next
npm ERR!   i18next@"^22.5.0" from the root project
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peer i18next@"^21.0.0" from i18next-cli-language-detector@1.1.7
npm ERR! node_modules/i18next-cli-language-detector
npm ERR!   i18next-cli-language-detector@"*" from the root project
. . .
```
I's a minor problem (I hope) but annoying. Could it be fixed?
Thanks!
mario

--- a/package.json
+++ b/package.json
@@ -69,6 +69,6 @@
     "typescript": "^5.0.4"
   },
   "peerDependencies": {
-    "i18next": "^21.0.0"
+    "i18next": "*"
   }
 }

